Source/WebKit/Platform/SharedMemory.h

```diff
@@ -26,6 +26,7 @@
 
 #pragma once
 
+#include <wtf/ArgumentCoder.h>
```

Comment:
Let’s use a forward declaration of the `ArgumentCoder` class template instead of an include, to keep build times in check.

Reply:
Fixed